src/actions/authorActions.js

```diff
@@ -0,0 +1,55 @@
+import * as types from './actionTypes';
+import authorApi from '../api/mockAuthorApi';
+import { beginAjaxCall, ajaxCallError } from './ajaxStatusActions';
+
+export function loadAuthorSuccess(authors) {
+  return { type: types.LOAD_AUTHORS_SUCCESS, authors };
+}
+
+export function createAuthorSuccess(author) {
+  return { type: types.CREATE_AUTHORS_SUCCESS, author };
+}
+
+export function updateAuthorSuccess(author) {
+  return { type: types.UPDATE_AUTHORS_SUCCESS, author };
+}
+
+export function deleteAuthorSuccess(authorId) {
+  return { type: types.DELETE_AUTHORS_SUCCESS, authorId };
+}
+
+export function loadAuthors() {
+  return function(dispatch) {
+    dispatch(beginAjaxCall());
+    return authorApi.getAllAuthors().then(authors => {
+      dispatch(loadAuthorSuccess(authors));
+    }).catch(error => {
+      throw(error);
```

Comment:
How are you handling this error? It is cool to throw errors, but they should be properly handled.
I would suggest you dispatch an error handling event and handle that on the UI.
since this is a small app you can just activate an error toast with the error message anytime the error action is called. This is also reoccuring across your code base